src/requestor/routes/manage.py

```diff
@@ -112,6 +112,12 @@ async def create_request(
     client = api_request.app.arborist_client
 
     if not data["policy_id"]:
+        # Check format of resource_paths (should have content after slash)
+        for p in data["resource_paths"]:
+            if not p.split("/")[1:]:
+                msg = f"Request creation failed. Resource path '{p}' does not have content after a forward slash ('/')."
+                raise_error(logger, msg, body)
```

Comment:
the validity of resource paths should be Arborist's concern, not Requestor's. It looks like Requestor would work fine no matter the format of the resource path. What error/issue are we actually trying to fix here?

Reply:
Requestor returns a 500 when receiving a request with a bad resource_path (without a slash). Arborist returns a 400.

Reply:
if this is when creating a new request, it sounds to me like we should catch that 400 from arborist, and return a 400 to the user after getting one from arborist. We shouldn't duplicate the input validation from the server into the client. Arborist probably returns a nice error message that Requestor can forward to the user

if the error happens later, during the automatic call to arborist when requests are approved, then it's a bit more tricky